hack/deploy-kubevirt.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 set -exo pipefail
 
+function getLatestPatchVersion {
+  local major_minors=$1
+  curl -s https://api.github.com/repos/kubevirt/kubevirt/releases?per_page=100 | grep .tag_name | grep ${major_minors} | sort -V | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs
```

Comment:
Note

after sort -V
```
    "tag_name": "v1.3.0",
    "tag_name": "v1.3.0-alpha.0",
    "tag_name": "v1.3.0-beta.0",
    "tag_name": "v1.3.0-rc.0",
    "tag_name": "v1.3.0-rc.1",
    "tag_name": "v1.3.0-rc.2",
    "tag_name": "v1.3.1",
```

theoretically it can take a RC, not a pure stable, it is fine on old releases, since usually they dont create RC after stable, on main it might select non "stable", but anyhow we need it just for releases

Reply:
Hey @oshoval I think we can proceed knowing this, right? can you re-approve it?